### ISSUE TITLE
DetailsList: Column header aria-describedby value is invalid

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#3732]
#### Describe the issue:
The value for aria-describedby for details list column header is invalid since it is not present in the dom.
Example on https://developer.microsoft.com/en-us/fabric#/components/detailslist

![image](https://user-images.githubusercontent.com/11937034/35034004-b4675a12-fb92-11e7-9669-33adeff80ecd.png)

#### Actual behavior:
The property is set to a tooltip element which is not visible.

![image](https://user-images.githubusercontent.com/11937034/35033991-a895482a-fb92-11e7-906e-71d0fb1db0f7.png)

#### Expected behavior:

The aria-describedby should have id of an element that is present in the dom.

**Comment by ThomasMichon** [comment]
I put out PR #4487 to address this.

**Comment by ThomasMichon** [comment]
`aria-describedby` is not redundant to `aria-labelledby`. `aria-labelledby` is a pointer to content which should *replace* the screen-readable content of the current element. `aria-describedby` is a pointer to content which augments the screen-readable content of the current element.

**Comment by oengusmacinog** [comment]
It looks to me like @ThomasMichon 's PR remedied the problem, so I am going to go ahead and close this issue
